tests/foreman/api/test_convert2rhel.py

```diff
@@ -265,9 +275,9 @@ def test_convert2rhel_oracle(target_sat, oracle, activation_key_rhel, version):
     )
     assert host_content['subscription_status'] == 0
 
-
+@pytest.mark.skip_if_open("BZ:2216907")
```

Comment:
Can we remove this as BZ is ON_QA already?
```suggestion

```

Reply:
Yes. Removed.